bots/task/__init__.py

```diff
@@ -338,27 +338,26 @@ def execute(*args):
     global verbose
     if verbose:
         sys.stderr.write("+ " + " ".join(args) + "\n")
-
-    # Github wants the OAuth token as the username and git will
-    # happily echo that back out.  So we censor all output.
-    def censored(text):
```

Comment:
We could leave the censoring in, just in case.

Reply:
I force-pushed to put back the `censored()`, and fix the typo in the commit message.